script.simkl/addon.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="script.simkl"
+       name="Simkl TV Tracker"
+       version="1.0.0"
+       provider-name="David Davó / Simkl">
+  <requires>
+    <import addon="xbmc.python" version="2.6"/>
+    <import addon="script.module.simplejson"/>
+    <import addon="script.module.xbmcutil"/>
+  </requires>
+  <extension point="xbmc.service" library="service.py" start="login" />
+  <extension point="xbmc.addon.metadata">
+    <summary lang="en_gb">Simkl Scrobbler with AI file detection system</summary>
+    <disclaimer lang="en_gb">Currently under development</disclaimer>
+    <description lang="en_gb">Automatically set Movies and TV Shows\Anime as watched on Simkl.com. Simkl knows what you're watching, tells you episodes you've missed, and connects you to what your friends are into.</description>
+    <platform>all</platform>
+    <license>GPLv3</license>
+    <email>david@ddavo.me</email>
+    <source>https://github.com/SIMKL/script.simkl</source>
+    <assets>
+      <icon>resources/icon.png</icon>
+      <fanart>resources/fanart.png</fanart>
+    </assets>
+    <language>en es</language>
```

Comment:
I think you should leave the languages tag blank. This is only applied to the content the plugin or script provides. Your addon.xml does not include a <provides> tag, so you're not providing media content with this script